doc_admin/AdminManual.adoc

```diff
@@ -121,12 +119,12 @@ Generally, the main aspects the Administrator has to pay attention to are:
 * Select *Detailed Configuration*
 * Select *Server Machine*
 * Select *Multifunctional Database*
-* Set a minimum of concurrent connection as the number of clients multiplied by two
+* Set a minimum of concurrent connection as twice the number of expected clients
```

Comment:
Putting my editor's hat on, even better wording would be:

Set the number of minimum concurrent connections to twice the number of expected clients

Reply:
Thank you David ! Please check _everything_ I'm proposing also ragarding documentation update and feel free to give suggestions and provide corrections :-)

"Set the number of concurrent connections to twice the number of expected clients" is even shorter and clearer.